spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/RibbonClientHttpRequestFactory.java

```diff
@@ -38,12 +34,8 @@
 
 	private final SpringClientFactory clientFactory;
 
-	private LoadBalancerClient loadBalancer;
```

Comment:
yeah, its a little confusing because the LoadBalancerCommand is doing similar things to the logic in the RibbonLoadBalancerClient. Is the "isSecure" flag there so that if some instances are secure and some are not, the URI will have the correct "http" or "https" when using that server?